diff -ruN a/lib60870-C/make/target_system.mk b/lib60870-C/make/target_system.mk
--- a/lib60870-C/make/target_system.mk	2023-01-19 14:11:41.188766182 +0100
+++ b/lib60870-C/make/target_system.mk	2023-01-19 14:11:08.180696014 +0100
@@ -146,8 +146,8 @@
 endif
 
 ifneq ($(TARGET), CLANG-CHECK)
-CC=$(TOOLCHAIN_PREFIX)gcc
-CPP=$(TOOLCHAIN_PREFIX)g++
+# CC=$(TOOLCHAIN_PREFIX)gcc
+# CPP=$(TOOLCHAIN_PREFIX)g++
 endif
 
 ifeq ($(TARGET), BSD)
diff -ruN a/lib60870-C/Makefile b/lib60870-C/Makefile
--- a/lib60870-C/Makefile	2023-01-19 14:09:05.009435603 +0100
+++ b/lib60870-C/Makefile	2023-01-19 14:12:10.157827760 +0100
@@ -143,9 +143,9 @@
 	$(CC) $(CFLAGS) -c $(LIB_INCLUDES) $(TEST_INCLUDES) $(OUTPUT_OPTION) $<
 	
 install:	$(LIB_NAME)
-	mkdir -p $(INSTALL_PREFIX)/include
+	mkdir -p $(INSTALL_PREFIX)/include/60870
 	mkdir -p $(INSTALL_PREFIX)/lib
-	cp $(LIB_API_HEADER_FILES) $(INSTALL_PREFIX)/include
+	cp $(LIB_API_HEADER_FILES) $(INSTALL_PREFIX)/include/60870
 	cp $(LIB_NAME) $(INSTALL_PREFIX)/lib
 
 clean:
